### PR TITLE
[RELENG-7422] 📝 Add tags documentation

### PR DESCRIPTION
**Pull request by gaspardmoindrot**

### CODE REVIEW

File: docs/tags.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Tags
+
+In addition to Grafana, we also have tags that are set on AWS and GCP to
+visualize data. These tags allow us to navigate their respective consoles
+and filter data using specific tags, providing more accurate figures
+compared to Grafana, which has an approximate margin of error of +/- 5%.
```
Comment (nootal) [inline docs/tags.md:6]: Why this margin of error?

File: docs/tags.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## AWS
+
+For AWS, we have the following tags (specified in the configuration file
+found in the `devinfra` repository):
```
Comment (nootal) [inline docs/tags.md:11]: Link to the specific file.

File: docs/tags.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+- Name of the runner
```
Comment (nootal) [inline docs/tags.md:13]: Gibe an example

File: docs/tags.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+- `lifecycle_autostop`: `no`
+- `lifecycle_autostart`: `no`
+- `map-migrated`: `mig42992`
+- `owner`: `ci`
+- `tool`: `runner-manager`
```
Comment (nootal) [inline docs/tags.md:18]: Better to put this list in a table format

File: docs/tags.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+## GCloud
```
Comment (nootal) [inline docs/tags.md:20]: Not `GCloud`. Find something that is official.

File: docs/tags.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+For GCP, we have the following tags (retrieve dynamic tags using the
+`gh_actions_exporter` repository via webhooks):
+
+- machine_type
```
Comment (nootal) [inline docs/tags.md:25]: Table format + some examples

**Review by nootal — CHANGES_REQUESTED** [review]

**Review by tcarmet — CHANGES_REQUESTED** [review]
Same issue encountered with your PR on the exporter.

We can't add information that is internal on a public repository.
The information we can find here can only be valid in the context of the project. Everything above it should not be mentioned.

**Comment by tcarmet** [comment]
We won't be merging this PR and going forward with this work.

**Pull request closed without merging**